HIMIKO/.htaccess

```
@@ -1,4 +1,4 @@
 Options +FollowSymLinks
 RewriteEngine on
 
-RewriteRule ^(.*)$ https://junjun7613.github.io/MicroKnowledge/$1 [R=302, L]
\ No newline at end of file
+RewriteRule ^himiko.owl$ https://junjun7613.github.io/MicroKnowledge/himiko.owl [R=302, L]
```

The original is fine except you need to remove the space in the options.
```suggestion
RewriteRule ^(.*)$ https://junjun7613.github.io/MicroKnowledge/$1 [R=302,L]
```

---

Thank you for your comment! I did remove the space from the original.